spec/govuk_error/configuration_spec.rb

```diff
@@ -11,20 +11,20 @@
     end
   end
 
-  describe ".should_capture" do
+  describe ".before_send" do
     let(:configuration) { GovukError::Configuration.new(Raven.configuration) }
 
     it "ignores errors if they happen in an environment we don't care about" do
       ClimateControl.modify SENTRY_CURRENT_ENV: "some-temporary-environment" do
         configuration.active_sentry_environments << "production"
-        expect(configuration.should_capture.call(StandardError.new)).to eq(false)
+        expect(configuration.before_send.call(StandardError.new)).to be_nil
```

Comment:
Ah, I see - because it needs to return `nil` to drop the event: https://docs.sentry.io/platforms/ruby/configuration/filtering/

Could you explain that in the commit message when you come to rebase this?

Reply:
Minor: Might be worth fleshing out the commit message slightly by linking to the URL above ^